lib/matplotlib/_mathtext.py

```diff
@@ -1740,71 +1763,57 @@ def set_names_and_parse_actions():
         p.overline         = Forward()
         p.overset          = Forward()
         p.placeable        = Forward()
-        p.required_group   = Forward()
+        p.req_group        = Forward()
```

Comment:
Is there an important reason for the abbreviation? I'd have gone with required_group and optional_group.

Reply:
This makes more of the definitions below fit in a single line (e.g. making the definitions of frac/dfrac/binom and overset/underset more clearly parallel each other.  I can undo that if you feel strongly about it, though.

Reply:
I prefer the full name, because that's more readable. I think wrapping is not too bad for the mentioned cases:
```
       p.binom <<= cmd(
           r"\binom", p.reqired_group("num") + p.reqired_group("den"))
```

Reply:
done